tsconfig.all.json

```diff
@@ -7,7 +7,8 @@
     "inlineSources": false,
     "noEmit": true,
     "outDir": "docs",
-    "sourceMap": false
+    "sourceMap": false,
+    "skipLibCheck": true
```

Comment:
please do not add this config, I noticed that `@rollup/plugin-typescript": "^6.1.0"` has a type problem, why do you need to update that library?

Reply:
The lib update came from a plugin, which accidentally updated all libs. Sorry - reverted